packages/venia-ui/lib/components/MiniCart/cartOptions.gql.js

```diff
@@ -4,7 +4,8 @@ import { CartPageFragment } from '../CartPage/cartPageFragments.gql';
 
 export const REMOVE_ITEM_MUTATION = gql`
     mutation remoteItem($cartId: String!, $itemId: Int!) {
```

Comment:
```suggestion
    mutation removeItem($cartId: String!, $itemId: Int!) {
```

I know you didn't modify this line but as long as we're in here 😝